README.md

```diff
@@ -39,6 +68,38 @@ search analytics.
   aa("setUserToken", "id-of-user");
 </script>
 ```
+<!-- prettier-ignore-end -->
+
+#### On Node.js
+
+Insights library can be used on the backend as a Node.js module. _(Node.js `>= 8.16.0` required)_
+
+```bash
+$ npm install search-insights
+or
+$ yarn add search-insights
+```
+
+```js
+const aa = require("search-insights);
+aa("init", {
+  appId: "APPLICATION_ID",
+  apiKey: "SEARCH_API_KEY"
+});
+```
+
+It exports the same `aa` interface.
+
+##### setUserToken
+
+On the Node.js environment, unlike the browser environment, `setUserToken` must be called before sending any event.
```

Comment:
For node I think most user will want to just include the userToken in the event itself. 
IMHO
- If you're seeding events, this is more practical
- if you're sending events from a route, this more practical too

```js
aa("clickedObjectIDs", {
   userToken: "xxx",
   /* ... */
})
```
We should at least document this other way of including the user token.